cuda_bindings/cuda/bindings/_lib/utils.pxi.in

```diff
@@ -135,7 +134,7 @@ cdef class _HelperInputVoidPtr:
         elif isinstance(ptr, (int)):
             # Easy run, user gave us an already configured void** address
             self._cptr = <void*><void_ptr>ptr
-        elif isinstance(ptr, (_driver.CUdeviceptr)):
+        elif isinstance(ptr, (_driver["CUdeviceptr"])):
```

Comment:
How about just deferring the (Python) import to `__cinit__`, and avoid building this dict?